lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+from __future__ import division
+import numpy as np
+import pandas as pd
+import warnings
+
+from lifelines.fitters import UnivariateFitter
+from lifelines.utils import _preprocess_inputs
+from lifelines.fitters.kaplan_meier_fitter import KaplanMeierFitter
+
+class AalenJohansenFitter(UnivariateFitter):
+    """
+    Class for fitting the Aalen-Johansen estimate for the cumulative incidence function in a competing 
+    risks framework.
+    
+    AalenJohansenFitter(alpha=0.95,jitter_level=0.00001,seed=None)
+    
+    Aalen-Johansen cannot deal with tied times. We can get around this by randomy jittering the event times 
+    slightly. This will be done automatically and generate a warning.
+    """
+    def fit(self,durations,event_observed,event_ind,timeline=None,entry=None,weights=None):
```

Comment:
from a pep8 point of view, can you add a space after each `,` - and not just on this line, but all lines of code here?